reconcile/test/test_utils_slack_api.py

```diff
@@ -168,6 +175,39 @@ def test_chat_post_message_missing_channel(slack_api):
         slack_api.client.chat_post_message('test')
 
 
+def test_chat_post_message_channel_not_found(mocker, slack_api):
+    slack_api.client.channel = 'test'
+    mock_join = mocker.patch('reconcile.utils.slack_api.SlackApi.join_channel')
+    nf_resp = new_slack_response({'ok': False, 'error': 'not_in_channel'})
+    slack_api.mock_slack_client.return_value.chat_postMessage.side_effect = \
+        [SlackApiError('error', nf_resp), None]
+    slack_api.client.chat_post_message('foo')
+    assert slack_api.mock_slack_client.return_value.chat_postMessage. \
+           call_count == 2
+    mock_join.assert_called_once()
+
+
+def test_chat_post_message_ok(slack_api):
+    slack_api.client.channel = 'test'
+    ok_resp = new_slack_response({'ok': True})
+    slack_api.mock_slack_client.return_value.chat_postMessage.side_effect = \
+        ok_resp
+    slack_api.client.chat_post_message('foo')
+    slack_api.mock_slack_client.return_value.chat_postMessage. \
+        assert_called_once
```

Comment:
This should be a function call? `assert_called_once()`?

Reply:
:facepalm: